h5p-bildetema/src/components/SubHeader/SubHeader.tsx

```diff
@@ -33,7 +35,8 @@ export const SubHeader: React.FC<SubHeaderProps> = ({
 
   const renderLeftMenu = (): JSX.Element => {
     const element = isWordView ? (
-      <span className={styles.toggle}>
+      <span className={styles.tools}>
+        <PrintButton label={printLabel} />
```

Comment:
Could we move the printLabel useL10n into PrintButton?

Reply:
Yesss, I was thinking about that too